Module4.1_Intro_to_Data_Structures_and_Algos/activities/Act4_Stacks/6.md

```diff
@@ -0,0 +1,56 @@
+<!--title={Using the Stack}-->
+
+<!--badges={Algorithms:5,Python:5}-->
```

Comment:
Make sure that this is building off of 5.md- it feels disconnected. Explain that you're automating the process and why it works!